grakn-graph/src/main/java/ai/grakn/graph/internal/RelationImpl.java

```diff
@@ -234,7 +201,6 @@ void cleanUp() {
      */
     @Override
     public void delete() {
-        scopes().forEach(this::deleteScope);
```

Comment:
Yes they do. You can delete the scope from a type. If you asking if the scope dies when the type dies then the answer to that is no.